org.eclipse.winery.topologymodeler.ui/src/app/services/error-handler.service.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { TestBed, inject } from '@angular/core/testing';
```

Comment:
We don't have tests yet, so please delete the auto generated ones.

Reply:
done